requirements.txt

```diff
@@ -1 +1,2 @@
-pygerduty==0.25
+pypd==0.1.0
+urllib3[secure]
```

Comment:
Yeah, just ignore that warning for now. Problem with Python version on Ubuntu 14.